test/client.js

```diff
@@ -39,7 +39,7 @@ setTimeout(function() {
           //console.log(JSON.stringify(base).length, 'reduced to', JSON.stringify(evt.value).length);
           prev = base;
         }
-        ws.send(JSON.stringify([evt.type, 'testid', evt.value]));
+        ws.send(JSON.stringify([evt.type, 'testid', evt.value, new Date(evt.time).getTime()]));
```

Comment:
I would add a comment explaining why we need this conversation

Reply:
can you ask early-2016-fippo? :-)

Reply:
(i am not even sure how the json file was generated...)